src/sonic_ax_impl/mibs/ieee802_1ab.py

```diff
@@ -37,8 +38,95 @@ class LLDPLocalChassis(int, Enum):
     lldp_loc_chassis_id = 2
     lldp_loc_sys_name = 3
     lldp_loc_sys_desc = 4
-    # *lldp_rem_sys_cap_supported = 5
-    # *lldp_rem_sys_cap_enabled = 6
+    lldp_loc_sys_cap_supported = 5
+    lldp_loc_sys_cap_enabled = 6
+
+
+class ManAddrConst(Enum):
```

Comment:
>Enum [](start = 19, length = 4)

Why oid is a Enum member?

Reply:
You could remove 'Enum' and make code cleaner. Mixing different types of constants are not like an enum.

---
In reply to: [205701552](https://github.com/Azure/sonic-snmpagent/pull/72#discussion_r205701552) [](ancestors = 205701552)